apps/desktop/src/lib/commit/CommitList.svelte

```diff
@@ -217,9 +242,9 @@
 					/>
 				{/each}
 
-				<CommitAction bottomBorder={hasRemoteCommits}>
+				<CommitAction bottomBorder={hasRemoteCommits} hideAction={$remoteCommits.length > 0}>
 					{#snippet lines()}
```

Comment:
If we want to hide the action, does it make more sense to not render the commit action at all, or to ask it to hide part of itself?

Reply:
It does 👍 

I initially chose this, in order to easily keep the virtual branch lines and right padding while hiding the button, but then we have two different levels at which we're hiding components, which is indeed not great.

I'll change this